Documentation/gettingstarted/cni-chaining-aws-cni.rst

```diff
@@ -0,0 +1,131 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+*******
+AWS-CNI
+*******
+
+This guide explains how to set up Cilium in combination with aws-cni. In this
+hybrid mode, the aws-cni plugin is responsible for setting up the virtual
+network devices as well as address allocation (IPAM) via ENI. After the initial
+networking is setup, the Cilium CNI plugin is called attach BPF programs to the
+network devices set up by aws-cni to enforce network policies, perform
+load-balancing, and encryption.
```

Comment:
Does this integration allows us to apply CiliumNetworkPolicy containing labels?

My understanding was that this is basically [Native Routing](https://cilium.readthedocs.io/en/v1.5/intro/#simple-networking), so we are still limited to use security group IDs and pod/service CIDRs only for CiliumNetworkPolicy rule conditions. But just wanted to confirm.

Reply:
@mumoshu This integration will allow you to use the full scope of CiliumNetworkPolicy including identity-based enforcement. This delegates the IPAM and network plumbing to aws-cni and uses Cilium to perform all policy and load-balancing aspects.